src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,9 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 - bet1 == 0 && result2 - bet2 == 0)
+                ? 2 : ((result1 < result2 && bet1 < bet2)
+                ? 1 : ((result1 > result2 && bet1 > bet2)
+                ? 1 : ((result1 == result2 && bet1 == bet2) ? 1 : 0)));
```

Comment:
too many conditions '?' try to simplify it

Reply:
done